### ISSUE TITLE
Serbian translation of the android and desktop app is very bad

### ISSUE THREAD

**Issue by DavidK004** [brave/brave-browser#35417]
## Description 
In the app both Cyrillic and Latin words are mixed (words written with Cyrillic alphabet and words written with Latin alphabet). In Serbian both Cyrillic and Latin alphabets are official but it is not grammatically correct to mix them.

## Steps to Reproduce

   1. Switch app to Serbian
   2. Go to settings
   3. See mixed Cyrillic and Latin

## Actual result:
Mixed alphabets

## Expected result:
Using only either Cyrillic or Latin alphabet

## Reproduces how often: 
Always there

## Desktop Brave version:
1.61.120 Chromium: 120.0.6099.234 (Званична верзија) (64-битни) 

## Android Device details:
   - Install type arm64 v8a
   - Device type Phone
   - Android version: 13

## Version/Channel Information:
All channels

- Can you reproduce this issue with the current release? Yes
- Can you reproduce this issue with the beta channel?  Yes
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by Susexe** [comment]
It's been this way for years. This [community post](https://community.brave.com/t/serbian-localization-chaos/305297) from almost three years ago is a good summary of the current state.

I started considering other browsers because it bothers me and looks very unprofessional. That's why I'm willing to help improve the Serbian translation by joining the localization team. Please contact me regarding this; I already sent you a message on Transifex.

**Comment by bsclifton** [comment]
cc: @AndyAnds265

**Comment by AndyAnds265** [comment]
@DavidK004, thank you for bringing this up. We are in the process of re-organizing our localization efforts. This will entail addressing multiple language related issues such as fonts and locale mapping. We will address this issue as well. I appreciate you bringing it to our attention

**Comment by Susexe** [comment]
@AndyAnds265, could you respond to my request? Aside from the issue of mixing scripts, the translation itself isn't very good. I can assist with it.

**Comment by Susexe** [comment]
@AndyAnds265, could you respond to my request? Aside from the issue of mixing scripts, the translation itself isn't very good. I can assist with it.

**Comment by AndyAnds265** [comment]
@Susexe - just providing an update. We're already live with both Serbian (Latin) and Serbian (Cyrillic) options on Brave Search. Please feel free to check them out at search.brave.com.

We are working actively on the fix for the Android app and will roll out as soon as we can :)

**Comment by Susexe** [comment]
@AndyAnds265 I added a comment on the issue above.

**Comment by kjozwiak** [comment]
Closing as the above was fixed via https://github.com/brave/brave-core/pull/27535 and merged into `1.76.x` via https://github.com/brave/brave-core/pull/27575. Labelled as `QA/No` as the above will be verified via https://github.com/brave/brave-browser/issues/43444.

**Comment by Susexe** [comment]
@AndyAnds265, thanks for your initiative!

If I understood the changes correctly, there won't be mixing of scripts any longer, which is great news. However, you default to Serbian Latin. The main script for Serbian in Chromium is Cyrillic, also Cyrillic is the official script used in Serbia. You mentioned in #39792 that you'll switch to Cyrillic as your base script for translations. Is that still the plan?

**Comment by Susexe** [comment]
I explained the perks of using Cyrillic as a base before, but it won't hurt to say it again: transliteration to Latin is 100% accurate, which would allow you to kill two birds with one stone.

**Comment by GKid94** [comment]
Fix Cyrillic too

**Comment by AndyAnds265** [comment]
@Susexe yes that is correct. Once this fix is rolled out we should have two distinct versions of Serbian. One will be the base, Cyrillic Serbian and the alternative Latin. They should match with Chromium options! Thank you for patience - I'm glad to hopefully have this all solved very soon. 

We've also made updates on Brave Search (Brave претрага) which should be rolled out this month. 

For any work we do moving forward, Cyrillic will be the base.

**Comment by AndyAnds265** [comment]
@GKid94 - I am not sure what you mean. Can you please explain further?

**Comment by Susexe** [comment]
@AndyAnds265, awesome!

Do you accept volunteer contributions to your translations? I'd like to help with Serbian, considering it can be improved quite a bit.

Thanks!

**Comment by GKid94** [comment]
> [@GKid94](https://github.com/GKid94) - I am not sure what you mean. Can you please explain further? If you are referencing the Serbian language, then it will be fixed soon.

Yes, thanks 👍
